.github/workflows/pr.yml

```diff
@@ -294,8 +291,10 @@ jobs:
         uses: golangci/golangci-lint-action@v3
         with:
           version: ${{ env.GOLANGCI_LINT_VERSION }}
-          working-directory: ${{ matrix.directory }}
+          working-directory: ${{ pkg }}
```

Comment:
Interpolation does not look right. Is pkg a variable?

Reply:
No, no it is not 😞.